apps/files/src/components/FilesApp.vue

```diff
@@ -236,7 +235,23 @@ export default {
       this.fileUploadProgress = progressTotal / this.inProgress.length
       return this.fileUploadProgress
     },
-
+    onSearch (searchQuery) {
+      this.searchQuery = searchQuery
+      // do not search for empty strings
+      if (!searchQuery) return
+      this.$client.files.search(searchQuery).then((searchedFiles) => {
```

Comment:
Dav props need to be addee here ...

Reply:
I wasn't sure about using them, because actually excluding them doesn't change anything.
I could omit `{http://owncloud.org/ns}size`, because in my case, the folder size (which is why we're using this prop, isn't it?) isn't visible anyway.
But I've used the default props from your test including the size prop.

We could provide some default list size via `state.config.defaultListSize` or something, but before then I don't know what's the best result size to use; So i prefer to use your hardcoded default value via providing `null` as second parameter.